dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -5044,6 +5055,17 @@ ReturnedColumn* buildAggregateColumn(Item* item, gp_walk_info& gwi)
                 ac->resultType(parm->resultType());
             }
         }
+        else if (bIsConst && hasDecimalConst && isAvg)
```

Comment:
aggregates with literal arguments will now have a proper scale and precision to process it later in RowAggregationUM::doNotNullConstantAggregate